modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonAbstractConnexionServerCodegen.java

```diff
@@ -594,6 +594,7 @@ private void addSecurityExtensions(OpenAPI openAPI) {
             }
         }
 
+        generateJSONSpecFile(objs);
```

Comment:
@jimschubert is line 598 `generateYAMLSpecFile(objs);` because we're now generating the spec in JSON?

Reply:
I think they should both be included for consistency across all generators, tbh. We're limiting what people can extend or customize when we don't have a consistent model bound to templates.

Reply:
As discussed, this is more like a proactive enhancement that the user didn't ask for. I'm ok with that and let's see if there are users asking for something similar in other generators.